css-media-queries/at-media.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Media Queries - At Media</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+
+    body {
+      min-height: 100vh;
+      background-color: palevioletred;
+    }
+
+    /* your code here 👇 */
+    @media (width > 780px) {
```

Comment:
I would recommend using min or max width instead of doing conditionals like this. It's generally best practice to use either min or max width.